infra/base-images/base-builder/Dockerfile

```diff
@@ -27,6 +27,27 @@ RUN apt-get install -y git \
     libblocksruntime-dev \
     libc6-dev-i386
 
+# Download and install the latest stable Go.
+ADD https://storage.googleapis.com/golang/getgo/installer_linux $SRC/
+
+# $SHELL is needed for the installer.
+ENV SHELL "bash"
+RUN chmod +x $SRC/installer_linux && \
```

Comment:
nit: why not add SHELL="bash" when calling the command itself, this removes lines 34 and 38.